rust/src/dcerpc/dcerpc.rs

```diff
@@ -186,6 +186,13 @@ pub struct DCERPCTransaction {
     pub tx_data: AppLayerTxData,
 }
 
+impl Transaction for DCERPCTransaction {
+    fn id(&self) -> u64 {
+        // need +1 to match state.tx_id
+        self.id + 1
```

Comment:
Is this correct? Other parsers aren't adding 1, whats the difference with this parser?

Reply:
Ah, this one doesn't incremement before assigning. It used to be that a tx_id of 0 cause issues elsewhere so we always started at 1.  Maybe this got fixed elsewhere, or was never an issue for Rust.  I think a comment is in order as its the odd one out.

Reply:
> Is this correct? Other parsers aren't adding 1, whats the difference with this parser?

The previous PR did not do it and CI fails (make check, and S-V dcerpc)

Is it good for you ?

Reply:
> > Is this correct? Other parsers aren't adding 1, whats the difference with this parser?
> 
> The previous PR did not do it and CI fails (make check, and S-V dcerpc)
> 
> Is it good for you ?

Yes.  Just required some extra review as its different than the others.